src/test/resources/Features/OCD-2049 & OCD-2822-productHistoryEye.feature

```diff
@@ -0,0 +1,70 @@
+Feature: OCD-2049 - Verify 'Eye' on listing details page works and displays Product History 
```

Comment:
1. Spaces in filenames are evil, always. Windows lets you get away with it, and generally OS's handle it okay, but as a rule, never have spaces in filenames
2. Look at the other feature files that have multiple tickets in them; none of them have the ticket number in the filename